available_images.md

```diff
@@ -100,6 +100,14 @@ E3 Framework Containers (EC2, ECS, and EKS support only)
 |TensorFlow 2.7.0   |inference	|Yes			|CPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.7.0-cpu-py38-ubuntu20.04-e3		|
 |TensorFlow 2.7.0   |inference	|Yes			|GPU 		| 3.8 (py38)			|763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.7.0-gpu-py38-cu112-ubuntu20.04-e3	|
 
```

Comment:
@arjkesh Should we modify the previous heading `E3 Framework Containers (EC2, ECS, and EKS support only)` to `E3 Framework X86  Containers (EC2, ECS, and EKS support only)`?

Reply:
I think if we do that here, we'll need to do that everywhere else. I think leaving as is should be fine, as below we've made a clear distinction for graviton